lib/index.ts

```diff
@@ -0,0 +1,81 @@
+import * as Unist from 'unist'
+
+const own = {}.hasOwnProperty
+function hasProperty(value: any, propertyName: string): boolean {
+  return own.call(value, propertyName)
+}
+
+function isNode(value: any): value is Unist.Node {
+  return hasProperty(value, 'position') || hasProperty(value, 'type')
+}
+
+function isPosition(value: any): value is Unist.Position {
+  return hasProperty(value, 'start') || hasProperty(value, 'end')
+}
+
+function isPoint(value: any): value is Unist.Point {
+  return hasProperty(value, 'line') || hasProperty(value, 'column')
+}
+
+export = stringify
+
+function stringify(
+  value: Unist.Node | Unist.Position | Unist.Point,
+): string | null {
+  /* Nothing. */
+  if (!value || typeof value !== 'object') {
+    return null
+  }
+
+  /* Node. */
+  if (isNode(value)) {
+    return stringifyPosition(value.position)
+  }
+
+  /* Position. */
+  if (isPosition(value)) {
+    return stringifyPosition(value)
+  }
+
+  /* Point. */
+  if (isPoint(value)) {
+    return stringifyPoint(value)
+  }
+
+  /* ? */
+  return null
+}
+
+function stringifyPoint(point: Unist.Point) {
```

Comment:
I renamed function name because tslint blames me that the argument name and the previous function name were same.